apps/files_external/lib/Lib/Storage/FTP.php

```diff
@@ -351,11 +351,12 @@ public function getDirectoryContent($directory): \Traversable {
 
 			$data = [];
 			$data['mimetype'] = $isDir ? FileInfo::MIMETYPE_FOLDER : $mimeTypeDetector->detectPath($name);
-			$timestamp = \DateTime::createFromFormat('YmdGis', $file['modify']);
-			$data['mtime'] = $timestamp ? $timestamp->getTimestamp() : (int) strtotime('YmdGis', $file['modify']);
-			if ($data['mtime'] === false) {
-				$data['mtime'] = time();
-			}
+			$modifyDate = \DateTime::createFromFormat('YmdGis', $file['modify']);
+			if ($modifyDate === false) {
+				$date['mtime'] = time();
```

Comment:
```suggestion
				$data['mtime'] = time();
```

Sorry. I made a typo :see_no_evil: